Builds/CMake/RippledCov.cmake

```diff
@@ -2,97 +2,39 @@
    coverage report target
 #]===================================================================]
 
-if (coverage)
-  if (is_clang)
-    if (APPLE)
-      execute_process (COMMAND xcrun -f llvm-profdata
-        OUTPUT_VARIABLE LLVM_PROFDATA
-        OUTPUT_STRIP_TRAILING_WHITESPACE)
-    else ()
-      find_program (LLVM_PROFDATA llvm-profdata)
-    endif ()
-    if (NOT LLVM_PROFDATA)
-      message (WARNING "unable to find llvm-profdata - skipping coverage_report target")
-    endif ()
-
-    if (APPLE)
-      execute_process (COMMAND xcrun -f llvm-cov
-        OUTPUT_VARIABLE LLVM_COV
-        OUTPUT_STRIP_TRAILING_WHITESPACE)
-    else ()
-      find_program (LLVM_COV llvm-cov)
-    endif ()
-    if (NOT LLVM_COV)
-      message (WARNING "unable to find llvm-cov - skipping coverage_report target")
-    endif ()
+ProcessorCount(PROCESSOR_COUNT)
 
-    set (extract_pattern "")
-    if (coverage_core_only)
-      set (extract_pattern "${CMAKE_CURRENT_SOURCE_DIR}/src/ripple/")
-    endif ()
+if (coverage)
+  if (DEFINED CODE_COVERAGE_TEST_PARALLELISM)
+    set(TEST_PARALLELISM ${CODE_COVERAGE_TEST_PARALLELISM})
+  else()
+    set(TEST_PARALLELISM ${PROCESSOR_COUNT})
+  endif()
```

Comment:
What is the difference between `TEST_PARALLELISM` and `CODE_COVERAGE_TEST_PARALLELISM`? If you're just setting a default value here, the best practice is to use `option()`.

Reply:
I need an actual default value, will use `set (... CACHE ...)`